drivers/display/ft8xx/ft8xx.c

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright (c) 2020 Hubert Miś
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/display/ft8xx/ft8xx.h>
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <device.h>
+#include <kernel.h>
+#include <logging/log.h>
+
+#include <drivers/display/ft8xx/ft8xx_copro.h>
+#include <drivers/display/ft8xx/ft8xx_dl.h>
+
+#include "ft8xx_common.h"
+#include "ft8xx_drv.h"
+#include "ft8xx_host_commands.h"
+#include "ft8xx_memory.h"
+
+#define DT_DRV_COMPAT ftdi_ft800
+
+#define LOG_MODULE_NAME ft8xx
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#define PCLK      DT_INST_PROP(0, pclk)
+#define PCLK_POL  DT_INST_PROP(0, pclk_pol)
+#define CSPREAD   DT_INST_PROP(0, cspread)
+#define SWIZZLE   DT_INST_PROP(0, swizzle)
+#define VSIZE     DT_INST_PROP(0, vsize)
+#define VOFFSET   DT_INST_PROP(0, voffset)
+#define VCYCLE    DT_INST_PROP(0, vcycle)
+#define VSYNC0    DT_INST_PROP(0, vsync0)
+#define VSYNC1    DT_INST_PROP(0, vsync1)
+#define HSIZE     DT_INST_PROP(0, hsize)
+#define HOFFSET   DT_INST_PROP(0, hoffset)
+#define HCYCLE    DT_INST_PROP(0, hcycle)
+#define HSYNC0    DT_INST_PROP(0, hsync0)
+#define HSYNC1    DT_INST_PROP(0, hsync1)
```

Comment:
I think it's better to collect DT properties into driver config structure, see e.g.  https://github.com/zephyrproject-rtos/zephyr/pull/30960